docs/VLSI/HAMMER.rst

```diff
@@ -34,25 +43,191 @@ The types of tools (in there hammer names) supported currently include:
 * sram_generator
 * pcb
 
-In order to configure your tool plugin of choice, you will need to set several configuration variables.
-First, you should select which specific tool you want to use by setting ``vlsi.core.<tool_type>_tool`` to the name of your tool.
-For example ``vlsi.core.par_tool: "innovus"``.
-You will also need to point hammer to the folder that contains your tool plugin by setting ``vlsi.core.<tool_type>_tool_path``.
-This directory should include a folder with the name of the tool as specified previously, which itself includes a python file ``__init__.py`` and a yaml file ``defaults.yml`` specifying the default values for any tool specific variables.
-In addition you can also customize the version of the tools you use by setting ``<tool_type>.<tool_name>.version`` to a tool specific string.
-Looking at the tools ``defaults.yml`` will inform you if there are other variables you would like to set for your use of this tool.
+Several configuration variables are needed to configure your tool plugin of choice.
+
+First, select which tool to use for each action by setting ``vlsi.core.<tool_type>_tool`` to the name of your tool, e.g. ``vlsi.core.par_tool: "innovus"``.
 
-The ``__init__.py`` file should contain a variable, ``tool``, that points to the class implementing this tools Hammer support.
-This class should be a subclass of ``Hammer<tool_type>Tool``, which will be a subclass of ``HammerTool``.
+Then, point Hammer to the folder that contains your tool plugin by setting ``vlsi.core.<tool_type>_tool_path``.
+This directory should include a folder with the name of the tool, which itself includes a python file ``__init__.py`` and a yaml file ``defaults.yml``. Customize the version of the tool by setting ``<tool_type>.<tool_name>.version`` to a tool specific string.
+
+The ``__init__.py`` file should contain a variable, ``tool``, that points to the class implementing this tool.
+This class should be a subclass of ``Hammer<tool_type>Tool``, which will be a subclass of ``HammerTool``. The class should implement methods for all the tool's steps.
+
+The ``defaults.yml`` file contains tool-specific configuration variables. The defaults may be overridden as necessary.
 
 Technology Plugins
 ==================
 
-Hammer supports separately managed plugins for different technologies.
+Hammer supports separately managed technology plugins to satisfy NDAs. You may be able to acquire access to certain pre-built technology plugins with permission from the technology vendor. Or, to build your own tech plugin, you need at least a ``<tech_name>.tech.json`` and ``defaults.yml``. An ``__init__.py`` is optional if there are any technology-specific methods or hooks to run.
+
+The `ASAP7 plugin <https://github.com/ucb-bar/hammer/tree/master/src/hammer-vlsi/technology/asap7>`__ is a good starting point for setting up a technology plugin. Refer to Hammer's documentation for the schema and setup instructions.
```

Comment:
It might be worth mentioning that it's a good start for setting up a tech plugin because it's an open-source example, and not because you would actually use it in a chip

Reply:
I think it goes without saying that you would not be able to tape out anything in ASAP7 since it's purely a predictive PDK.

Reply:
You'd be surprised how many people in the architecture world would require that to be said. It doesn't hurt us to write that sentence, right?